grails-app/controllers/org/pih/warehouse/product/ProductController.groovy

```diff
@@ -674,6 +676,26 @@ class ProductController {
         }
     }
 
+    def showAssociatedProducts = { StockCardCommand cmd ->
+        def product = Product.get(params.id)
+        def location = Location.get(session?.warehouse?.id)
+
+        def associations = product?.associations
+        def products = product?.associatedProducts() as List
+        def quantityAvailableMap = [:]
+        if (!products.isEmpty()) {
+            quantityAvailableMap = productAvailabilityService.getQuantityAvailableToPromiseByProduct(location, products)
```

Comment:
Why you added this? Was it specifically requested to also have a quantity available to promise?